src/Microsoft.Health.Dicom.Api/Web/SeekableStreamConverter.cs

```diff
@@ -56,31 +56,28 @@ public async Task<Stream> ConvertAsync(Stream stream, CancellationToken cancella
 
         private static class AspNetCoreTempDirectory
         {
-            private static string _tempDirectory;
+            private static string s_tempDirectory;
```

Comment:
This is the default prefix for static members. How do people feel about `"s_"` prefixes?

Reply:
Based on my discussions with folks, we're ok with this!